### DISCUSSION TITLE
Build Time

### DISCUSSION THREAD

**Discussion by akil-rails** [vuejs/vitepress#1260] [Q&A]
My current build with ~25 pages is taking 46 secs. Is this the benchmark or is there a way to improve this speed?

**Comment by brc-dd** [comment]
Vue.js' site is much larger than 25 pages and it takes <40s to build. VitePress' site is ~25 pages and it takes <8s to build. I guess that will vary depending on your content and what additional plugins/components are there. But 46s seems long for 25 pages unless they have very long content.

**Comment by akil-rails** [comment]
The pages are short, but there are images and code snippets and some plugins; Is there a way to find out what is causing the delay?
